deploy/backplane/lpsre/acm/acm/02-acm-lpsre-admins-project.ClusterRole.yaml

```diff
@@ -0,0 +1,926 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: backplane-acm-lpsre-admins-project
+rules:
+- apiGroups:
+  - ''
+  - action.open-cluster-management.io
+  - addon.open-cluster-management.io
+  - agent.open-cluster-management.io
+  - app.k8s.io
+  - apps.open-cluster-management.io
+  - argoproj.io
+  - authorization.k8s.io
+  - certificates.k8s.io
+  - cluster.open-cluster-management.io
+  - clusterregistry.k8s.io
+  - compliance.mcm.ibm.com
+  - config.openshift.io
+  - coordination.k8s.io
+  - discovery.open-cluster-management.io
+  - hive.openshift.io
+  - hiveinternal.openshift.io
+  - imageregistry.open-cluster-management.io
+  - internal.open-cluster-management.io
+  - inventory.open-cluster-management.io
+  - mcm.ibm.com
+  - multicloud.ibm.com
+  - operator.open-cluster-management.io
+  - policy.open-cluster-management.io
+  - proxy.open-cluster-management.io
+  - rbac.authorization.k8s.io
+  - register.open-cluster-management.io
+  - search.open-cluster-management.io
+  - submarineraddon.open-cluster-management.io
+  - view.open-cluster-management.io
+  - work.open-cluster-management.io
+  resources:
+  - applicationrelationships
+  - applicationrelationships/status
+  - applications
+  - applications/status
+  - certificatesigningrequests
+  - certificatesigningrequests/approval
+  - channels
+  - channels/status
+  - clusterclaims
+  - clustercurators
+  - clusterdeployments
+  - clustermanagementaddons
+  - clustermanagers
+  - clusterpools
+  - clusterrolebindings
+  - clusterroles
+  - clusterstatuses/aggregator
+  - clustersyncs
+  - clusterversions
+  - compliances
+  - configmaps
+  - deployableoverrides
+  - deployableoverrides/status
+  - deployables
+  - deployables/status
+  - discoveredclusters
+  - discoveryconfigs
+  - endpointconfigs
+  - endpoints
+  - events
+  - helmrepos
+  - helmrepos/status
+  - klusterletaddonconfigs
+  - leases
+  - machinepools
+  - managedclusteractions
+  - managedclusteractions/status
+  - managedclusteraddons
+  - managedclusteraddons/status
+  - managedclusterimageregistries
+  - managedclusterimageregistries/status
+  - managedclusterinfos
+  - managedclusterinfos/status
+  - managedclusters
+  - managedclusters/accept
+  - managedclusters/status
+  - managedclustersetbindings
+  - managedclustersets
+  - managedclustersets/bind
+  - managedclustersets/join
+  - managedclustersets/status
+  - managedclusterviews
+  - managedclusterviews/status
+  - manifestworks
+  - manifestworks/status
+  - namespaces
+  - placementbindings
+  - placementdecisions
+  - placementdecisions/status
+  - placementrules
+  - placementrules/status
+  - placements
+  - pods
+  - pods/log
+  - policies
+  - policies/status
+  - policyautomations
+  - policysets
+  - policysets/status
+  - rolebindings
+  - roles
+  - searchcustomizations
+  - secrets
```

Comment:
What secret data is needed by LPSRE?  Generally recommend we do not provide secret access without justification.  Note that list/watch also permit reading the secret.